bundles/core/src/main/java/com/adobe/cq/wcm/core/components/services/form/FormPostService.java

```diff
@@ -0,0 +1,38 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ ~ Copyright 2020 Adobe
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+package com.adobe.cq.wcm.core.components.services.form;
+
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.api.SlingHttpServletResponse;
+
+/**
+ * A service that posting form data to a remote service.
+ *
+ * @since com.adobe.cq.wcm.core.components.services.form 1.0.0
+ */
+public interface FormPostService {
+
+    /**
+     * Send form data to a remote servcie.
+     *
+     * @param request the {@link SlingHttpServletRequest}
+     * @param response the {@link SlingHttpServletResponse}
+     * @return true if the remote request was successful, otherwise false
+     *
+     * @since com.adobe.cq.wcm.core.components.services.form 1.0.0
+     */
+    boolean sendFormData(SlingHttpServletRequest request, SlingHttpServletResponse response);
```

Comment:
I'd rename it `forwardFormData` as it is forwarding form data to an external service.

Reply:
When forwarding the form data to the external service, I would expect following arguments to also be present in the method signature:
- the form data, e.g. `JSONObject formData`. We could introduce a FormUtils class to retrieve the form data from the request
- the configuration of the external service, e.g. `String endPointUrl`. We could have a FormUtils method to retrieve it.